### ISSUE TITLE
Validation loss='nan' while training

### ISSUE THREAD

**Issue by vanikanjirangat** [yuanluo/text_gcn_tutorial#1]
I am getting validation loss='nan' while training.

Tensor("graphconvolution_2/SparseTensorDenseMatMul/SparseTensorDenseMatMul:0", shape=(None, 3), dtype=float32)
Epoch: 0001 train_loss= 1.09876 train_acc= 0.24752 val_loss= nan val_acc= 0.19512 time= 2.22297
Epoch: 0002 train_loss= 1.08244 train_acc= 0.50226 val_loss= nan val_acc= 0.19512 time= 0.18762
Epoch: 0003 train_loss= 1.05605 train_acc= 0.50858 val_loss= nan val_acc= 0.19512 time= 0.17869
Epoch: 0004 train_loss= 1.01898 train_acc= 0.50136 val_loss= nan val_acc= 0.19512 time= 0.17944

Appreciate your help!
